contents/blog/decouple-deployment-from-release.md

```diff
@@ -0,0 +1,114 @@
+---
+title: Why you should decouple deployment from release (and how) 
+date: 2023-09-11
+author: ["ian-vanagas"]
+showTitle: true
+rootpage: /blog
+sidebar: Blog
+hideAnchor: true
+featuredImage: ../images/blog/green-blog-image.jpg
+featuredImageType: full
+category: Engineering
+tags:
+ - Guides
+ - Explainers
+---
+
+Releasing a big feature can be nerve-racking. When deploying it, you don't know if it will work in production, break your app, or cause issues. Without the right processes in place, the safety of your releases is a big question mark.
+
+A best practice for improving this is decoupling deployment and release. What does this mean, though? This post goes over the differences between the two, why you should decouple them, examples, and how to get started.
+
+## What are the differences between deployment and release?
+
+The process of delivering code starts with writing and testing code locally. After completing a piece of code, it often goes through reviews and more tests before integrating with the existing codebase. 
+
+Once integrated, the new code deploys. Deployment means getting code from a local machine or repository into production. Your app or build containing the new code becomes available on production infrastructure.
+
+A release is when new code is put into use or made available to users. By default, as soon as code is deployed, it is automatically released. This is because there is nothing preventing users or systems from accessing the new code available on production infrastructure.
```

Comment:
```suggestion
A release is when the new code is made available to users. By default, as soon as code is deployed, it is automatically released. This is because there is nothing preventing users or systems from accessing the new code available on production infrastructure.
```